README.md

```diff
@@ -67,6 +67,27 @@ You can now use the high-fidelity tools to tweak your CSS and inspect network ca
 
 ![Microsoft Edge Tools - Demo](basic_usage.gif)
 
+
+#### Setting up your project to show live changes in the extension
```

Comment:
The README has gotten big enough that I think it would greatly benefit from a table of contents.  We don't necessarily need to do it in this PR, but if we don't let's open an issue on it.

Reply:
Yes, we should also take a look at some of the other parts in there that are tough to understand.